### PR TITLE
[Platform.sh]: ElasticSearch support

### PR DESCRIPTION
**Pull request by adamwojs**
> JIRA: - 

### Description 

Added configuration template for ElasticSearch. 

_Note for reviewers: please review PR carefully. I'm pretty new to platform.sh ;-)_

### TODO

*   [x] Confirm that ES 7.7 is available

### CODE REVIEW

File: .platform.app.yaml [hunk 1]
```diff
@@ -161,8 +162,11 @@ hooks:
         ##php bin/console doctrine:migrations:migrate --no-interaction --allow-no-migration
         ##php bin/console kaliop:migration:migrate --no-interaction --no-debug
 
-        # When using Solr, there are two cases where you'll need to rebuild indexes:
-        # - When Solr / search configuration changes
+        # Configure ElasticSearch
+        ##php bin/console ezplatform:elasticsearch:put-index-template
```
Comment (micszo) [inline .platform.app.yaml:166]: There's an issue here. This gets executed on platform.sh after `composer ezplatform-install` which is followed by `ezplatform:reindex`.
Possibly this command should be moved before `HTTPCACHE_PURGE_TYPE="local" php -d memory_limit=-1 'which composer' ezplatform-install` on line 148. (that worked for QA) \cc @mnocon

**Review by micszo — CHANGES_REQUESTED** [review]
ES 7.7 is the latest available version: https://docs.platform.sh/configuration/services/elasticsearch.html#supported-versions.

**Comment by adamwojs** [comment]
PR updated according to @micszo code review suggestions and rebased against 3.1 branch.

**Review by micszo — APPROVED** [review]
This order works. 👍

**Comment by lserwatka** [comment]
@adamwojs could you merge it up and sideways?

**Pull request merged**